src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -783,7 +783,7 @@ public int CompareTo(String strB)
         }
 
         [Pure]
-        internal bool EndsWith(char value) {
+        public Boolean EndsWith(char value) {
```

Comment:
The same here, I was not sure if I should use the more verbose syntax or not. I'll change it. I also prefer this way. Thanks